api/AudioBuffer.json

```diff
@@ -0,0 +1,659 @@
+{
+  "api": {
+    "AudioBuffer": {
+      "AudioBuffer": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioBuffer/AudioBuffer",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "58"
+              },
+              {
```

Comment:
If the context parameter has been actually removed in 57, I would put this as a note to the standard version.
If it is still correct, I would make it a subfeature of the construction.